packages/mux-uploader/src/mux-uploader-drop.ts

```diff
@@ -0,0 +1,168 @@
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+  /* These styles simulate a user passing these via props. Until
+     that's implemented, we just hardcode it for non-full-screen demo purposes. (TD).
+  */
+  .dropzone {
```

Comment:
Yeah I wouldn't worry too much about solutioning right now, since we probably want to strike a balance between trying to anticipate use cases vs. users informing us about pain in their use cases. Also, fwiw, the `stylePropsToString` is more for handling the `React` -> Web Component translation, so folks can do things in a fairly "idiomatic to React" way even though, "under the hood," we are frequently using Web Components.